server/src/main/scala/drt/users/KeyCloakGroups.scala

```diff
@@ -34,4 +36,20 @@ case class KeyCloakGroups(groups: List[KeyCloakGroup]) {
     })
     Future.sequence(eventualUsersWithGroupsByGroup).map(_.flatten)
   }
-}
\ No newline at end of file
+
+  def usersWithGroupsByUser(groups: List[KeyCloakGroup]) =
```

Comment:
Can we add a type annotation here? It's not easy to figure out what's being returned